### PR TITLE
correctly use seconds for caching header

### PR DESCRIPTION
**Pull request by atomfrede**
closes https://github.com/jhipster/generator-jhipster/issues/19102

### CODE REVIEW

**Comment by atomfrede** [comment]
Will check the test. Executing it with idea worked fine 🤔

**Comment by ge65cer** [comment]
Thanks for the fix, don't forget hat the `Expires` header probably expects milliseconds to [calculate the future date](https://github.com/jhipster/jhipster-bom/blob/main/jhipster-framework/src/main/java/tech/jhipster/web/filter/CachingHttpHeadersFilter.java#L75). 🙂

**Review by ge65cer — APPROVED** [review]
Thanks for the fix, looks good to me! 👍

File: jhipster-framework/src/main/java/tech/jhipster/web/filter/CachingHttpHeadersFilter.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public class CachingHttpHeadersFilter implements Filter {
     /** Constant <code>DEFAULT_DAYS_TO_LIVE=1461</code> */
     public static final int DEFAULT_DAYS_TO_LIVE = 1461; // 4 years
     /** Constant <code>DEFAULT_SECONDS_TO_LIVE=TimeUnit.DAYS.toMillis(DEFAULT_DAYS_TO_LIVE)</code> */
```
Comment (DanielFran) [inline jhipster-framework/src/main/java/tech/jhipster/web/filter/CachingHttpHeadersFilter.java:36]:
```suggestion
    /** Constant <code>DEFAULT_SECONDS_TO_LIVE=TimeUnit.DAYS.toSeconds(DEFAULT_DAYS_TO_LIVE)</code> */
```

**Review by DanielFran — APPROVED** [review]

**Pull request merged**